### DISCUSSION TITLE
When is the emit() event going to be available on the stately ai editor?

### DISCUSSION THREAD

**Discussion by yoguib** [statelyai/xstate#4912] [Q&A] [answered]
When is the emit() event going to be available on the stately ai editor? Thanks for your help

**Comment by davidkpiano** [comment]
Today!

**Reply by davidkpiano** [reply, marked as answer]
This is now available:

<img width="547" alt="CleanShot 2024-05-28 at 09 09 17@2x" src="https://github.com/statelyai/xstate/assets/1093738/083d09b7-6ccc-4bdd-b988-e8b4f1ce52b4">

**Reply by yoguib** [reply]
Legend!
